src/printing.py

```diff
@@ -12,6 +12,30 @@
 # y, and z:
 # x is 10, y is 2.25, z is "I like turtles!"
```

Comment:
You are missing the correct formatting for this! Right now you are printing them all separately.